chsdi/models/vector/uvek.py

```diff
@@ -1710,6 +1710,18 @@ class HindernisbegrenzungsflaechenPerimeter(Base, Vector):
 register('ch.bazl.hindernisbegrenzungsflaechen-perimeter', HindernisbegrenzungsflaechenPerimeter)
 
 
+class IntrinsischesBodenrisikoSora(Base, Vector):
+    __tablename__ = 'intrinsisches_bodenrisiko_sora'
+    __table_args__ = ({'schema': 'bazl', 'autoload': False})
+    __template__ = 'templates/htmlpopup/bazl_intrinsisches_bodenrisiko.mako'
+    __bodId__ = 'ch.bazl.intrinsisches-bodenrisiko_sora'
+    id = Column('id', Integer, primary_key=True)
+    density_pop_km2 = Column('density_pop_km2', Numeric)
+    the_geom = Column(Geometry2D)
+
+register('ch.bazl.intrinsisches-bodenrisiko_sora', IntrinsischesBodenrisikoSora)
```

Comment:
@rebert when do I put:
```
register(IntrinsischesBodenrisikoSora.__bodId__, IntrinsischesBodenrisikoSora)
```
instead of
```
register('ch.bazl.intrinsisches-bodenrisiko_sora', IntrinsischesBodenrisikoSora)
```
It should be the same no?

Reply:
yes, the result is the same :-)

Reply:
It is fixed here in this PR #4211